lib/helper/helper.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:google_mobile_ads/google_mobile_ads.dart';
+
+class Helper {
```

Comment:
you can use the helper class for creating objects if needed and load all the adds here.
Name it more specifically. No need to mention helper in the name